src/app/space/create/deployments/apps/deployment-details.component.less

```diff
@@ -1,5 +1,9 @@
 @import (reference) '../../../../../assets/stylesheets/shared/osio.less';
 
+pfng-chart-sparkline {
```

Comment:
errrr .. I think this line is better off as: 
`.deployment-chart pfng-chart-sparkline {`
to make it more specific?